### PR TITLE
Initial Version of Library

### PR DESCRIPTION
**Pull request by makermelissa**
Here is the initial working version of the library.

### CODE REVIEW

**Review by ladyada — APPROVED** [review]
approved, will merge once bitcoin example is updated to 60 seconds!

**Review by ladyada — APPROVED** [review]

File: adafruit_magtag/fakerequests.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: 2017 Scott Shawcroft, written for Adafruit Industries
+# SPDX-FileCopyrightText: Copyright (c) 2020 Melissa LeBlanc-Williams for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+"""
+`adafruit_magtag.fakerequests`
```
Comment (tannewt) [inline adafruit_magtag/fakerequests.py:6]: Why not factor this out into a separate library? It looks like a copy and paste out of other portal libraries.

Comment (makermelissa) [reply adafruit_magtag/fakerequests.py:6]: That's the eventual plan ;)

File: examples/bitcoin_demo.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+timestamp = None
+
+while True:
+    if not timestamp or (time.monotonic() - timestamp) > 60:  # once every 60 seconds...
+        try:
+            value = magtag.fetch()
+            print("Response is", value)
+        except (ValueError, RuntimeError) as e:
+            print("Some error occured, retrying! -", e)
+        timestamp = time.monotonic()
```
Comment (tannewt) [inline examples/bitcoin_demo.py:46]:
```python
else:
    time.sleep(time.monotonic() - timestamp)
```

**Review by tannewt — COMMENTED** [review]
Please please please start factoring out common code between the portal libraries instead of copying. Doing so will make them all more maintainable and easier to build upon. network, graphics and wifi_module are all things that could be used separately from MagTag. peripherals is what I'd expect the MagTag library to consist of. It encapsulates *just* the things that make the board unique.

File: examples/bitcoin_demo.py (hunk 2 above)
Comment (makermelissa) [reply examples/bitcoin_demo.py:46]: Fair enough. I'll create an issue for this.

File: adafruit_magtag/fakerequests.py (hunk 1 above)
Comment (tannewt) [reply adafruit_magtag/fakerequests.py:6]: When will it be done? This is our third portal library.

Comment (makermelissa) [reply adafruit_magtag/fakerequests.py:6]: I'll get to it soon. I have some stuff to clear off my plate first. You can keep an eye on issue #12.

**Pull request merged**